scripts/purge_sqs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Set the endpoint URL
+ENDPOINT_URL="http://sqs.us-east-1.localhost.localstack.cloud:4566"
+
+# Define an array of queue names
+QUEUES=(
+    "madara_orchestrator_snos_job_processing_queue"
+    "madara_orchestrator_snos_job_verification_queue"
+    "madara_orchestrator_proving_job_processing_queue"
```

Comment:
I think there are too many places now where we are managing this list. can we make a single source of truth ?

Reply:
Fair enough, creating an issue for this